### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by RolandAmini**
I have made sure the content of the "About myself" section is aligned with flexbox.

In order to lay out the elements in "Works" section I have used CSS Grid and Flexbox combined. Also Used CSS Grid in the main container that contains all of the cards and used Flexbox to lay out an individual card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,44 +9,216 @@
...
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
```
Comment (souad988) [inline index.html:86]: Great work was done so far by using semantic tags and descriptive classNames, only some small styling adjustements needed : 

1. the title needs to be bolder and the size needs to be increased
2. the cards horizontal margin needs to be increased to match the given design

image (14)
image (15)

File: index.html [hunk 2]
```diff
@@ -9,44 +9,216 @@
...
+      <div class="card">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
```
Comment (souad988) [inline index.html:158]: image (20)
your button are well styled they look exactly like the design but still missing the hover and press effects

File: index.html [hunk 3]
```diff
@@ -9,44 +9,216 @@
...
+      <!-- card -->
+      <div class="card">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
```
Comment (souad988) [inline index.html]: Kindly add the missing image to this card

image (19)

File: index.html (hunk 2 above)
Comment (souad988) [inline index.html]: your button are well styled they look exactly like the design but still missing the hover and press effects 

image (20)

File: index.html [hunk 4]
```diff
@@ -9,44 +9,216 @@
...
+      <li class="skill skill-about">Database Management</li>
+      <li class="skill skill-about">Version Control</li>
+      <li class="skill skill-about">CLI</li>
+      <li class="skill skill-about">Web Development</li>
+      <li class="skill ">API Design</li>
+      
+    </ul>
+    <button class="btn btn-skill">See Project</button>
+  </div>
+</div>
```
Comment (souad988) [inline index.html]: Kindly fix this card styling to match the design

image (21)

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @RolandAmini , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights
Good PR description
Good use of html semantic tags
Professional README

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @RolandAmini , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights
Good PR description
Good use of html semantic tags
Professional README

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RolandAmini** [comment]
thank you I have done all changes

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hi @RolandAmini, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉 
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Incorporates feedback from previous code reviews.
✅ Nice mobile design.

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Review by shella12 — APPROVED** [review]
Hi @RolandAmini, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great design implementation ✔️ 
- README is professional 👍 
- Implemented all required changes 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**